tox.ini

```diff
@@ -6,6 +6,7 @@ envlist =
     cov-combine
     cov-report
     mypy
+    mypy_untyped_defs
```

Comment:
There is no CI step which runs `tox` sans arguments.

Do you want to add this to the lint job?

https://github.com/globus/globus-cli/blob/5775c9fb3c2bfad8bb7077bda516d513f0ac6c01/.github/workflows/build.yaml#L10

Without it being in CI, I think the check is a weaker protection.

Reply:
Right you are! Update on its way.